great_expectations/experimental/datasources/data_asset/data_connector/filesystem_data_connector.py

```diff
@@ -36,8 +36,8 @@ def __init__(
         self,
         datasource_name: str,
         data_asset_name: str,
-        base_directory: pathlib.Path,
         regex: re.Pattern,
+        base_directory: pathlib.Path,
```

Comment:
I would try to resist the urge to make these minor stylistic changes unless there is a compelling reason.
It makes it harder to follow the git history.

Reply:
@Kilo59 If you re referring to the changed order, then it had to be done for better clarity and comprehension of the story that this code tells (this early in the development, unfortunately an earlier contribution did not get it right).  Having said that, I agree that we should have a justification for all changes, including cosmetic ones, such as this.  Thanks.